excessive_rolling_update.yml

```diff
@@ -54,6 +54,13 @@
         - current_version.stdout_lines[0].replace(' ','').split(':')[1] < "v2.0.1"
         - tidb_version >= "v2.1.0" or tidb_version == "latest"
 
+- name: Pre-check for configuration
+  hosts: pd_servers[0]
+  tags:
+    - check_config
+  roles:
+    - check_config
```

Comment:
Did this run after deploy new binary

Reply:
This check run before deployment. If errors are checked after deployment, the configuration needs to be reupdated after modification

Reply:
maybe it have some error because the old version did not have the check command

Reply:
rest LGTM

Reply:
This may happen, we will reconsider.

Reply:
This does not happen, this check is to distribute the new binary and configuration files to a temporary directory to check